docs/monitoring/spm-faq.md

```diff
@@ -21,11 +21,11 @@ See [integrations](/integration). You can also send [custom metrics](custom-metr
 
 **What are the various configuration files present in spm-client package?**
 
-The spm-client package consists Infra and App Agent. The configuration files these these agents are:
+The spm-client package contains Infra and App Agent. The configuration files of these agents are:
 
 1. `/opt/spm/properties/st-agent.yml` - Contains Infra Agent related configurations.
-2. `/opt/spm/spm-monitor/conf/spm-monitor-config-<YOUR_TOKEN>-<JVM_NAME>.properties` - One file for each of the App Agent setup in this host.
-3. `/opt/spm/properties/agent.properties` - Configuration that are common to both Infra and App Agents.
+2. `/opt/spm/spm-monitor/conf/spm-monitor-config-<YOUR_TOKEN>-<JVM_NAME>.properties` - One file for each of the App Agent setup on this host.
```

Comment:
> Of course, that "<-JVM NAME> part may not be there at all if we are monitoring some non-JVM thing, like MySQL

It is always there, in case of MySQL value is just `default`.

> side-note: @bsmid with tags becoming proper first class citizens, would it be possible to axe this whole "jvm" thing and just treat it like the rest of the tags?

There are few issues with that. E.g., smaller ones like - we would lose compatibility with old configs and fair amount of refactoring would be needed. More importantly though, it is a way in which we ensure uniqueness of agent instance. Say you want to monitor 4 Solr nodes on the same machine (they are naturally part of the same app). The only way currently to have unique config for each of them is to have jvmname as a part of config file name. So, we'd need to rethink things, possibly do some major refactoring. Otherwise, yeah, I wanted it to be gone too for some time.